dist/ExtUtils-CBuilder/lib/ExtUtils/CBuilder/Base.pm

```diff
@@ -51,24 +52,31 @@ sub new {
 
     ## If the path is just "cc", fileparse returns $ccpath as "./"
     $ccpath = "" if $self->{config}{cc} =~ /^\Q$ccbase$ccsfx\E$/;
-      
+
     foreach my $cxx (@{$cc2cxx{$ccbase}}) {
-      my $cxx1 = File::Spec->catfile( $ccpath, $cxx . $ccsfx);
 
-      if( can_run( $cxx1 ) ) {
-        $self->{config}{cxx} = $cxx1;
-	last;
-      }
-      my $cxx2 = $cxx . $ccsfx;
+      if ( $ccpath ) {
+          my $cxx1 = File::Spec->catfile( $ccpath, $cxx . $ccsfx);
+
+          if( can_run( $cxx1 ) ) {
+              $self->{config}{cxx} = $cxx1;
+	            last;
+          }
```

Comment:
The indentation here doesn't match the rest of the file, and has added literal tabs mixed with the spaces.

Reply:
I didn't actually change that from the original, the literal tab was there before. But let me fix that.